hibernate-core/src/main/java/org/hibernate/metamodel/mapping/internal/ToOneAttributeMapping.java

```diff
@@ -89,6 +89,7 @@
 
 	private final String sqlAliasStem;
 	private final boolean isNullable;
+	private boolean isIgnoreNotFound;
```

Comment:
Make `final`. Also, IMO we should ensure that isNullable is set to `true` if `isIgnoreNotFound` is true.